pxtsim/runtime.ts

```diff
@@ -93,6 +93,11 @@ namespace pxsim {
 
             this.serialOutBuffer += s;
             if (/\n/.test(this.serialOutBuffer) || this.serialOutBuffer.length > SERIAL_BUFFER_LENGTH) {
+                /** 
```

Comment:
Still needed?

Reply:
We should load smoothie on demand. In most cases, it won't be needed.

Reply:
Deleted commented-out code.

Post-beta TODO: load smoothie on demand